src/docs/community/innersourcing.mdx

```diff
@@ -16,12 +16,7 @@ Alternatively, you may [send us an email](/community/contactus).
 
 Developers can also choose to add a feature by creating a pull request on our [GitHub](https://github.com/brightlayer-ui). Please check out our [Contribution Guidelines](https://github.com/brightlayer-ui/.github/blob/master/CONTRIBUTING.md) before you get started.
 
-We build our code examples using GitHub and use several online tools for sharing them. These tools allow you to import GitHub repositories and make the code runnable and editable in an online IDE.
-
--   For Angular and React projects we use [Code Sandbox](https://codesandbox.io).
--   For React Native we use [Snack](https://snack.expo.io/). Snack requires that your project is built using the [Expo CLI](https://docs.expo.io/workflow/expo-cli/).
-
-If you have not used these tools before, they're a great way to play with code and make sharing code snippets much easier. We recommend that you try them out when submitting code samples to speed up the process.
+We build our code examples using GitHub, and use several online tools, such as [Code Sandbox](https://codesandbox.io) and [StackBlitz](https://stackblitz.com/) for sharing them. These tools allow you to import GitHub repositories and make the code runnable and editable in an online IDE. If you have not used these tools before, they're a great way to play with code and make sharing code snippets much easier. We recommend that you try them out when submitting code samples to speed up the process.
```

Comment:
> We build our code examples using GitHub**,** and use several online tools

This is a very small thing but it's bothering me a bit; could you remove the comma after "Github"?   It just seems like an unnecessary pause.